### PR TITLE
chore: small fixes

### PR DESCRIPTION
**Pull request by asafshen**
## Description
small TS related fixes

### CODE REVIEW

File: packages/sdks/nextjs-sdk/CHANGELOG.md
Comment (nirgur) [inline packages/sdks/nextjs-sdk/CHANGELOG.md:1]: Are those changes intentional?

File: packages/sdks/nextjs-sdk/examples/app-router/middleware.ts [hunk 1]
```diff
@@ -2,8 +2,7 @@ import { authMiddleware } from '@descope/nextjs-sdk/server';
 
 export default authMiddleware({
 	projectId: process.env.NEXT_PUBLIC_DESCOPE_PROJECT_ID,
-	baseUrl: process.env.NEXT_PUBLIC_DESCOPE_BASE_URL,
-	baseStaticUrl: process.env.NEXT_PUBLIC_DESCOPE_BASE_STATIC_URL
```
Comment (nirgur) [inline packages/sdks/nextjs-sdk/examples/app-router/middleware.ts:6]: Is this not needed?

File: packages/sdks/react-sdk/README.md [hunk 2]
```diff
@@ -503,6 +503,8 @@ See bellow for an `.env` file template with more information.
 
 ### Run Example
 
+Note: Due to an issue with react-sdk tsconfig, you need to remove `"examples"` from the `exclude` field in the `tsconfig.json` file in the root of the project before running the example.
```
Comment (nirgur) [inline packages/sdks/react-sdk/README.md:506]: 👍 
We need to extract the sample app into a new package

**Review by nirgur — APPROVED** [review]
2 small comments

File: packages/sdks/nextjs-sdk/CHANGELOG.md
Comment (asafshen) [reply packages/sdks/nextjs-sdk/CHANGELOG.md:1]: Yes, next middleware doesn't get static base url

**Pull request merged**